charts/gotenberg/values.yaml

```
@@ -74,6 +74,7 @@ autoscaling:
   minReplicas: 1
   maxReplicas: 100
   behavior: {}
+  customMetrics: {}
```

```suggestion
  extraMetrics: {}
```

---

In fact this probably be an array of extra metrics.

```suggestion
  extraMetrics: []
```